stage1_fly/enter/main.go

```diff
@@ -47,6 +55,33 @@ func getRootDir(pid string) (string, error) {
 	return os.Readlink(rootLink)
 }
 
+func getPodManifest() (*schema.PodManifest, error) {
+	f, err := os.Open("pod")
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+
+	pmb, err := ioutil.ReadAll(f)
+
+	if err != nil {
+		return nil, errwrap.Wrap(errors.New("error reading pod manifest"), err)
+	}
+	pm := &schema.PodManifest{}
+	if err = pm.UnmarshalJSON(pmb); err != nil {
+		return nil, errwrap.Wrap(errors.New("invalid pod manifest"), err)
+	}
+	return pm, nil
+}
+
+func getRuntimeApp(pm *schema.PodManifest) (*schema.RuntimeApp, error) {
+	if len(pm.Apps) != 1 {
+		return nil, fmt.Errorf("flavor %q only supports 1 application per Pod for now", flavor)
```

Comment:
I don't think this needs to be generic on flavor name, as this directory/subtree is specific to fly.